cmake/clang-format.cmake

```
@@ -0,0 +1,33 @@
+find_program(PYTHON_EXECUTABLE python3 python)
+
+if(NOT PYTHON_EXECUTABLE)
+  message(WARNING "Python not found. Skipping lint and format checks.")
+else()
+  if(DEFINED ENV{IS_CI})
+    set(IS_CI $ENV{IS_CI})
+  else()
+    set(IS_CI OFF)
+  endif()
+
+  if(IS_CI)
```

We don't need to set `IS_CI` cmake variable, since it's not used anywhere else.

```suggestion
  if(DEFINED ENV{IS_CI})
```